build.gradle

```diff
@@ -48,7 +48,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:$android_plugin_version"
+        classpath "com.android.tools.build:gradle:3.2.1"
```

Comment:
@ubuntudroid Can you remove android_plugin_version since it's not used now?

Reply:
Uh, sorry, happened while auto-updating.